src/model/application.model.ts

```diff
@@ -34,6 +35,8 @@ export interface ApplicationData {
     has_sold_land?: string;
     is_secure_register?: string;
     who_is_registering?: string;
+    companyProfile?: ICompanyDetails;
```

Comment:
Should the session not contain the entity model - the company profile is what we read in from the public api?